d3rlpy/algos/torch/cql_impl.py

```diff
@@ -300,3 +299,22 @@ def _compute_conservative_loss(
         data_values = (self._q_func(obs_t) * one_hot).sum(dim=1, keepdim=True)
 
         return (logsumexp - data_values).mean()
+
+    @train_api
+    @torch_api(scaler_targets=["obs_t", "obs_tpn"])
```

Comment:
If you take `TorchMiniBatch` object, it's already scaled. So you don't have to explicitly specify this.
```
@torch_api()
```

Reply:
Makes sense - thanks for the feedback! I have made the change in a new commit